packages/react/src/Card/Card.module.css

```diff
@@ -19,12 +19,36 @@
     'action action';
   flex: auto;
   grid-template-rows: auto auto auto auto 1fr;
+  position: relative;
+}
+
+.Card:focus-within {
+  outline: var(--brand-borderWidth-thick) solid var(--brand-color-focus);
 }
 
 .Card:hover {
   text-decoration: none;
 }
 
+.Card__link {
+  color: currentColor;
+  text-decoration: none;
+}
+
+/* The focus outline will appear around the entire card, not just the title */
+.Card__link:focus {
+  outline: none;
+}
+
+.Card__link::before {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
```

Comment:
I think we could use `inset: 0;`  as a shorthand here.

Reply:
Oh right, I always forget about that property. Thanks!!